src/components/Sum.jsx

```diff
@@ -1 +1,12 @@
+import React from 'react';
 // export function Sum() {}
+export const Sum = ({ a, b }) => (
+  <p>
+    {`Sum of ${+a} and ${+b} is ${+a + +b}`}
```

Comment:
use prop types property to specify correct types for component arguments instead of using `+` to transforming them to number